components/script/dom/bytelengthqueuingstrategy.rs

```diff
@@ -70,32 +68,20 @@ impl ByteLengthQueuingStrategyMethods for ByteLengthQueuingStrategy {
         // Step 1. Let steps be the following steps, given chunk
         // Note: See ByteLengthQueuingStrategySize instead.
 
-        unsafe {
-            // Step 2. Let F be !CreateBuiltinFunction(steps, 1, "size", « »,
-            // globalObject’s relevant Realm).
-            let raw_fun = JS_NewFunction(
-                *cx,
-                Some(byte_length_queuing_strategy_size),
-                1,
-                0,
-                b"size\0".as_ptr() as *const c_char,
-            );
-            assert!(!raw_fun.is_null());
-
-            // Step 3. Set globalObject’s byte length queuing strategy size function to
-            // a Function that represents a reference to F,
-            // with callback context equal to globalObject’s relevant settings object.
-            let fun_obj = JS_GetFunctionObject(raw_fun);
-            let fun = Function::new(cx, fun_obj);
-            global.set_byte_length_queuing_strategy_size(fun.clone());
-            Ok(fun)
-        }
+        // Step 2. Let F be !CreateBuiltinFunction(steps, 1, "size", « »,
+        // globalObject’s relevant Realm).
+        let fun = native_fn!(byte_length_queuing_strategy_size, b"size\0", 1, 0);
```

Comment:
Any reason why it can't be `c"size"`?

Reply:
@sagudev used  C-string literals, please check again